mock/mock.ts

```diff
@@ -27,6 +27,7 @@ export interface Config {
   Mode: string;
 }
 export function NewContext(conf: Config) {
+  console.log("Keploy running in: ",process.env.KEPLOY_MODE," mode");
```

Comment:
Please place this log statement after [this](https://github.com/keploy/typescript-sdk/blob/10ff4ebc89a3e99fecb4040cf37c6ba274556d50/mock/mock.ts#L59) because here we are considering mode either from env variable or conf.mode parameter. Also log value of "mode" in log statement instead of process.env